lua/nvim-web-devicons/icons-default.lua

```diff
@@ -2132,6 +2252,18 @@ local icons_by_file_extension = {
     cterm_color = "124",
     name = "Out",
   },
+  ["part"] = {
+    icon = "󰇚",
+    color = "#44cda8",
+    cterm_color = "43",
+    name = "Part",
+  },
+  ["patch"] = {
+    icon = "",
```

Comment:
Nice

Reply:
Should I leave it like that or it's better to use the +- icon? I think fits better.

Reply:
I like that one.